client/src/annoMatrix/annoMatrix.ts

```diff
@@ -599,7 +614,7 @@ Return cache keys for columns associated with this query.  May return
   _gcField(
     field: Field,
     isHot: boolean,
-    pinnedColumns: string[] | number[] | [undefined]
+    pinnedColumns: string[] | number[]
```

Comment:
This should return the new Dataframe LabelArray type (see my comment on the definition of WhereCacheColumnLabels)

Reply:
Updated [annoMatrix.ts:_gcField](https://github.com/chanzuckerberg/cellxgene/blob/mim/annoMatrix/client/src/annoMatrix/annoMatrix.ts#L613).